object.go

```diff
@@ -759,14 +756,24 @@ func (obs *obs) UpdateMeta(name string, meta *ObjectMeta) error {
 	if err != nil {
 		return err
 	}
-	// Copy new meta
-	info.ObjectMeta = *meta
-	mm := NewMsg(fmt.Sprintf(objMetaPreTmpl, obs.name, sanitizeName(meta.Name)))
-	mm.Data, err = json.Marshal(info)
-	if err != nil {
+
```

Comment:
Derek - is it an error to update meta for a deleted object?

Reply:
I think so.